### PR TITLE
[CI:DOCS] Update remote_client.md

### PR DESCRIPTION
**Pull request by m4z**
Explicitly include copy-and-paste code line for `ssh-copy-id` because people like myself will easily forget this part and (repeatedly) overlook it in the text. :blush:

Signed-off-by: Christopher 'm4z' Holm <them4z@gmail.com>

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by m4z** [comment]
Hmm, I'm using the GitHub WebUI… Can i sign-off by just changing my initial comment above?

**Comment by rhatdan** [comment]
You can try by adding an empty line followed by something like:

    Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

**Comment by m4z** [comment]
> You can try by [...]

Thanks, I've (hopefully) done that.

**Comment by rhatdan** [comment]
This has to be on the commit message.

File: docs/tutorials/remote_client.md [hunk 1]
```diff
@@ -62,7 +62,10 @@ Remote Podman uses SSH to communicate between the client and server. The remote
 ```
 ssh-keygen -t ed25519
 ```
-Your public key by default should be in your home directory under ~/.ssh/id_ed25519.pub. You then need to copy the contents of id_ed25519.pub and append it into  ~/.ssh/authorized_keys on the Linux  server. You can automate this using ssh-copy-id.
+Your public key by default should be in your home directory under ~/.ssh/id_ed25519.pub. You then need to copy the contents of id_ed25519.pub and append it into  ~/.ssh/authorized_keys on the Linux  server. You can automate this using ssh-copy-id:
```
Comment (TomSweeneyRedHat) [inline docs/tutorials/remote_client.md:65]: It's not always going to be `id_ed25519.pub` right?  Perhaps we can make it a generic `{MyPublicKey}` and then list an example for the first one ala:

```suggestion
Your public key by default should be in your home directory under ~/.ssh/{MyPublicKey}, where {MyPublicKey} would be named something like `id_ed25519.pub`.  You then need to copy the contents of id_ed25519.pub and append it into  ~/.ssh/authorized_keys on the Linux  server. You can automate this using ssh-copy-id:
```

Comment (TomSweeneyRedHat) [reply docs/tutorials/remote_client.md:65]: and then change id_ed25519.pub to {MyPublicKey} in the other places it's currently occuring.

**Pull request closed without merging**